roles/programming-langs/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+# tasks file for programming-langs
+
+- name: Install Ruby packages
+  apt: name={{ ruby_packages }} state=latest
```

Comment:
Unless @ripleymj disagrees, can we switch these to use the longer-form syntax? It'll make it easier to add new arguments in the future if necessary. I plan to open an issue to have us convert all the `apt` stuff to use the longer syntax so that it looks like every other task. It'd be nice to have this PR merged with the longer format.

```yaml
  apt:
    name: "{{ ruby_packages }}"
    state: "latest"
```